emcc.py

```diff
@@ -2999,12 +3002,12 @@ def worker_js_script(proxy_worker_filename):
   return web_gl_client_src + '\n' + proxy_client_src
 
 
-def system_js_libraries_setting_str(libs, lib_dirs, settings_changes, input_files):
+def process_libraries(libs, lib_dirs, settings_changes, input_files):
   libraries = []
 
   # Find library files
   for i, lib in libs:
-    logger.debug('looking for library "%s"', lib)
+    logger.debug('looking for library "%s" (%s)', lib)
```

Comment:
This looks wrong somehow.. am I missing something?

Reply:
Yeah, thanks, that's odd - seems like it should error. I guess we have no EMCC_DEBUG using test with interesting enough library usage.